nanoFramework.System.Net.Http/Http/HttpClientHandler.cs

```diff
@@ -348,7 +348,8 @@ HttpResponseMessage CreateResponseMessage(HttpWebResponse wr, HttpRequestMessage
 
         bool GetConnectionKeepAlive(HttpRequestHeaders headers)
         {
-            return headers.Connection.Equals("Keep-Alive");
+            // In theory, the value should be lower case but it can with upper case.
+            return headers.Connection.ToLower().Equals("keep-alive");
```

Comment:
Wont this fail when `headers.Connection` is `null`?
Speaking of which there should be a default for Connection header being added in the default headers collection isn't it?

Reply:
The code was existing ;-) And headers seem not to be null.

Reply:
Yes. I'm aware of that. 🤪 Was asking for the `Connection` header specifically. 
I've just looked at the code and it won't ever be null, so all good on this.